x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/assistant_settings_management.tsx

```diff
@@ -232,65 +235,80 @@ export const AssistantSettingsManagement: React.FC<Props> = React.memo(
       resetSettings();
       setHasPendingChanges(false);
     }, [resetSettings]);
-
     return (
       <>
-        <EuiPageTemplate.Header pageTitle="Settings" tabs={tabs} paddingSize="none" />
+        <EuiPageTemplate.Header
+          pageTitle={i18n.SECURITY_AI_SETTINGS}
+          tabs={tabs}
+          paddingSize="none"
+        />
         <EuiPageTemplate.Section
-          paddingSize="l"
+          paddingSize="none"
           css={css`
             padding-left: 0;
             padding-right: 0;
+            padding-top: ${euiTheme.base * 0.75}px;
+            padding-bottom: ${euiTheme.base * 0.75}px;
           `}
         >
+          {selectedSettingsTab === CONNECTORS_TAB && (
+            <ConnectorsSettingsManagement navigateToApp={navigateToApp} />
```

Comment:
We can get `navigateToApp` from within this component via context, right? Is there any reason to get it outside and make it be passed by props?

Reply:
Yeah saw that I called `useAssistantContext` in this component, I'll remove this prop here.